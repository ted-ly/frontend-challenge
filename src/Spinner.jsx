import React from 'react';

export default function Spinner() {
    return (
        <div className="Spinner">
          <div className="Spinner__bounce1"></div>
          <div className="Spinner__bounce2"></div>
          <div className="Spinner__bounce3"></div>
        </div>
      );
}