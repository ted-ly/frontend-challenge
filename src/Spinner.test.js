import renderer from 'react-test-renderer';
import Spinner from './Spinner';

describe('Spinner', () => {
    it('should match snapshot', () => {
        const component = renderer.create(<Spinner />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});