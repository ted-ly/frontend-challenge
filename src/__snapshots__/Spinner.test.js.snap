// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match snapshot 1`] = `
<div
  className="Spinner"
>
  <div
    className="Spinner__bounce1"
  />
  <div
    className="Spinner__bounce2"
  />
  <div
    className="Spinner__bounce3"
  />
</div>
`;
